setup-production.adoc

```diff
@@ -731,7 +881,15 @@ source .venv/bin/activate
 cp ~/taiga-protected/env.sample ~/taiga-protected/.env
 ----
 
+[source, bash]
+----
+MAX_AGE=360
+TAIGA_SECRET_KEY="taiga-back-secret-key"
```

Comment:
According to the real [env.sample](https://github.com/kaleidos-ventures/taiga-protected/blob/taiga_in_subpath/env.sample), the variable name `TAIGA_SECRET_KEY` is wrong. It should be `SECRET_KEY`.

SECRET_KEY=taiga-back-secret-key
MAX_AGE=300
TAIGA_SUBPATH=